server/proxy/uninstall.go

```diff
@@ -35,43 +43,99 @@ func (p *Proxy) UninstallApp(client mmclient.Client, sessionID string, cc *apps.
 	}
 
 	if message == "" {
-		message = fmt.Sprintf("Uninstalled %s", app.DisplayName)
+		message = p.conf.I18N().LocalizeWithConfig(loc, &i18n.LocalizeConfig{
+			DefaultMessage: &i18n.Message{
+				ID:    "apps.uninstall.default",
+				Other: "Uninstalled {{.AppID}}",
+			},
+			TemplateData: map[string]string{
+				"AppID": app.DisplayName,
```

Comment:
Maybe out of scope of this PR: Should `DisplayName` or `AppID` be used in the command outputs? The code base it not consistent on that.

Reply:
I feel app id is more unique, and therefore should be used for most of these. Not sure though. I don't mind changing these in this PR if we get to a decision.

Reply:
@levb Do you have thoughts on this? I'm 2/5 that `AppID` is a technical necessity, which we shouldn't show to the user if possible.